composer.json

```diff
@@ -12,6 +12,7 @@
     "php": ">=7.2",
     "fond-of-spryker/company-user-reference": "dev-master",
     "fond-of-spryker/company-user-reference-quote-connector": "dev-master",
+    "fond-of-codeception/spryker": "^1.0.0",
```

Comment:
I guess this hard dependency should be removed

Reply:
Done